public/app/features/dashboard-scene/scene/VizPanelSubHeader.tsx

```diff
@@ -127,7 +127,8 @@ export class VizPanelSubHeader extends SceneObjectBase<VizPanelSubHeaderState> {
     }
 
     if (this._adHocVar) {
-      this._adHocVar?.subscribeToState((n, p) => {
+      this._adHocSub?.unsubscribe();
+      this._adHocSub = this._adHocVar?.subscribeToState((n, p) => {
```

Comment:
I missed keeping track of these subs and dealing with them manually and addressed that here